### PR TITLE
Add action description and branding

### PR DESCRIPTION
**Pull request by H00N24**

### CODE REVIEW

**Review by fnast9 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Currently, it's an alpha release.
 
 ## Usage
 
-To use this action, add the `Certora Run GitHub Application` to the repository and add
+To use this action, add the [Certora Run Application] to the repository and add
```
[the remark this answers is not in the document]
Comment (H00N24) [reply README.md:8]: Fixed.

File: README.md [hunk 2]
```diff
@@ -68,6 +68,9 @@ besides the permissions, the action requires the following secrets:
 - `server` - Server to run the tests on (optional). Default is `production`.
 - `solc-remove-version-prefix` - Prefix to remove from the Solidity version (optional).
 - `job-name` - Name of the job (optional).
+- `install-java` - Install Java for type checking (optional). Default is `true`.
+- `compilation-steps-only` - Only compile the contracts (optional). Default is `false`.
```
[the remark this answers is not in the document]
Comment (H00N24) [reply README.md]: Fixed.

File: action.yml [hunk 3]
```diff
@@ -1,7 +1,12 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/SchemaStore/schemastore/refs/heads/master/src/schemas/json/github-action.json
 name: Certora Run Action
 description: |-
-  Test
+  Certora Run Action submits prover jobs to Certora Cloud for verification and returns a
+  report with aggreaged results.
```
[the remark this answers is not in the document]
Comment (H00N24) [reply action.yml]: Fixed.

**Review by H00N24 — COMMENTED** [review]
Thanks for the review, all of them should be fixed now.

**Pull request merged**